misc/docker-integration-test.py

```diff
@@ -65,9 +65,10 @@ def ignore_dirs(walked_dir, _):
     # based on a tag or SHA, and we want to build based on what was vendored in
     dockerfile_addition = ("\n"
         "RUN set -x && "
-        "GOPATH=$(pwd)/vendor/src/github.com/docker/notary/Godeps/_workspace:$GOPATH "
+        "export GO15VENDOREXPERIMENT=1 && "
```

Comment:
If this is set, is it necessary to have the vendor dir set in the GOPATH? (on the basis we don't support Go < 1.5)